com.chenyifaer.FaForever.yml

```diff
@@ -0,0 +1,82 @@
+id: com.chenyifaer.FaForever
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: faforever
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-download
+
+modules:
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.44.0.tar.xz
+        sha256: c66530e41ba59b1edba4ee89ef20b2188e273bed0497e95084729e3cfbe30c87
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      # taken from https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/5a241420c2ee85cc5080dd7cf7ba3049fd80fcf1/elements/world/webkit2gtk-4.0.bst
+      - -DPORT=GTK
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DUSE_GTK4=OFF
+      - -DUSE_LIBBACKTRACE=OFF
+      - -DUSE_SOUP2=ON
+    modules:
+      - shared-modules/libsoup/libsoup-2.4.json
+
+      - name: unifdef
+        no-autogen: true
+        make-install-args:
+          - prefix=${FLATPAK_DEST}
+        sources:
+          - type: archive
+            url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+            sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+        cleanup:
+          - '*'
+
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  # for some controller mods (see https://github.com/isXander/Controlify/issues/31)
+  - shared-modules/libusb/libusb.json
+
+  - name: faforever
+    buildsystem: simple
+    only-arches:
+    - x86_64
+    build-commands:
+    - ar -x *.deb
+    - tar -xf data.tar.gz
+    - install -Dm755 usr/bin/fa-forever /app/bin/faforever
+    - mkdir -p /app/share/icons/hicolor/scalable/apps
+    - install -Dm644 com.chenyifaer.FaForever.png /app/share/icons/hicolor/scalable/apps/com.chenyifaer.FaForever.png
+    - install -Dm644 com.chenyifaer.FaForever.desktop /app/share/applications/com.chenyifaer.FaForever.desktop
+    - install -Dm644 com.chenyifaer.FaForever.appdata.xml /app/share/appdata/com.chenyifaer.FaForever.appdata.xml
+    sources:
+      - type: file
+        only-arches:
+        - x86_64
+        url: https://github.com/cyf/faforever/releases/download/v0.0.0+22/fa-forever_0.0.0+22_amd64.deb
```

Comment:
it's written "open source" in the appstream file. so why not building from source and offering aarch64 support at the same time.

Reply:
flatpak/flatpak-builder-tools#383 blocked me.